pages/account.vue

```diff
@@ -246,4 +259,33 @@ async function handleCustomerServiceLinkButtonClick() {
   crisp?.do('chat:show')
   crisp?.do('chat:open')
 }
+
+async function handleClearReaderCacheButtonClick() {
+  useLogEvent('clear_reader_cache')
+
+  if (!window.caches) return
```

Comment:
For `!window.caches`, hide the button, or at least show a fake success message? 
Otherwise button would seem broken

Reply:
Updated